src/systems/ec/implementations/react/components/site-header-core/src/SiteHeaderCore.jsx

```diff
@@ -1,13 +1,17 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
 
 import Icon from '@ecl/ec-react-component-icon';
+import Link from '@ecl/ec-react-component-link';
 import SearchForm from '@ecl/ec-react-component-search-form';
 import { LanguageListOverlay } from '@ecl/ec-react-component-language-list';
 
 const SiteHeaderCore = ({
   logo,
+  logged,
```

Comment:
Variable communicating a state would be probably more easier to figure as `isLogged` without the documentation.

Reply:
true, but I kept the naming that was in place for other site headers.
If we change the parameter name, that would be considered as a breaking change (unless ke keep the two parameters)